metaseq/service/constants.py

```diff
@@ -56,3 +56,6 @@
     f"--max-tokens {BATCH_SIZE * MAX_SEQ_LEN}",
     "/tmp",  # required "data" argument.
 ]
+
+# Optional arg overrides which influence model loading during inference
+ARG_OVERRIDES = {}
```

Comment:
Maybe name this to be a slightly more specific "INFERENCE_ARG_OVERRIDES" or something, rather than "ARG_OVERRIDES", so it'll be slightly less confusing reading through code what this is?

Reply:
Good point! Let me make this change